### PR TITLE
Tracom tweak 2.0

### PR DESCRIPTION
**Pull request by elgeonmb**
https://github.com/PolarisSS13/Polaris/pull/4021

HERE'S HOPING

### CODE REVIEW

**Comment by Anewbe** [comment]
Compile your code before you commit, it, please. That last one would have been caught there, instead of by Travis. That means I don't have to laugh at you, and you can keep your shame secret.

**Comment by elgeonmb** [comment]
Got all them git books open as we speak.

File: code/modules/busy_space/organizations.dm [hunk 1]
```diff
@@ -18,7 +18,6 @@
 		"Glory",
 		"Axiom",
 		"Eternal",
-		"Icarus",
```
Comment (Anewbe) [inline code/modules/busy_space/organizations.dm]: But whyyyyyyy?

File: code/modules/busy_space/organizations.dm [hunk 2]
```diff
@@ -79,7 +78,6 @@
 		"NSS Exodus in Nyx",
 		"NCS Northern Star in Vir",
 		"NCS Southern Cross in Vir",
-		"NDV Icarus in Nyx",
```
Comment (Anewbe) [inline code/modules/busy_space/organizations.dm]: See above

File: code/modules/busy_space/organizations.dm [hunk 3]
```diff
@@ -113,7 +111,9 @@
 
 	ship_prefixes = list("HTV" = "freight", "HTV" = "munitions resupply")
 	destination_names = list(
-		"a SolGov dockyard on Luna"
+		"a SolGov dockyard on Luna",
+		"a Navy outpost in the Almach Rim",
+		"a Navy outpost on the Moghes border"
```
Comment (Anewbe) [inline code/modules/busy_space/organizations.dm]: Fleet, not Navy.

File: code/modules/busy_space/organizations.dm [hunk 4]
```diff
@@ -113,7 +111,9 @@
 
 	ship_prefixes = list("HTV" = "freight", "HTV" = "munitions resupply")
 	destination_names = list(
-		"a SolGov dockyard on Luna"
+		"a SolGov dockyard on Luna",
+		"a Navy outpost in the Almach Rim",
```
Comment (Anewbe) [inline code/modules/busy_space/organizations.dm]: Fleet, not Navy.

**Comment by elgeonmb** [comment]
a) thought I said Fleet IDFK

b) See the last PR: Nanotrasen Does Not Like Icarus

**Pull request merged**